idaes/generic_models/properties/core/generic/utility.py

```diff
@@ -32,7 +32,7 @@ class StateIndex(Enum):
     apparent = 2
 
 
-class GenericPropertyPackageError(PropertyPackageError):
+class GenericPropertyPackageError(PropertyPackageError, IdaesError):
```

Comment:
You don't need to list `IdaesError` here, as it is already in the class hierarchy through `PorpertyPackageError`